README.md

```diff
@@ -113,6 +113,18 @@ client = Restforce.new(username: 'foo',
                        api_version: '38.0')
 ```
 
+#### JWT Bearer Token
+
+If you prefer to use a [JWT Bearer Token](https://developer.salesforce.com/page/Digging_Deeper_into_OAuth_2.0_on_Force.com#Obtaining_an_Access_Token_using_a_JWT_Bearer_Token) to authenticate:
+
+```ruby
+client = Restforce.new(username: 'foo',
+                       client_id: 'client_id',
+                       instance_url: 'instance_url',
+                       jwt_key: '/path/to/private/key',
```

Comment:
Would it be too much to ask that the `jwt_key` refer to the key as a string instead of a path to the key? I'm using Heroku and I put private keys in environment variables to avoid checking the key into git. (Maybe I should be doing something else?)

Reply:
It would be good to have. But, for now I have used a hack for this - 

```ruby
    def private_key_file
      private_key_file = Tempfile.new('tempfile')
      private_key_file.write(ENV['RSA_PRIVATE_KEY'])
      private_key_file.rewind
      private_key_file.close
      private_key_file
    end
```